quizbizz/src/pages/Homepage/HeroSection/herosection.css

```diff
@@ -0,0 +1,55 @@
+.hero-section-container{
+    background-color: var(--background-color-dark1);
+    border-radius: 2rem;
+    margin: 3rem 2rem;
+    box-shadow: 0px 3px 8px var(--tertiary-color);
+}
+
+/* Hero Section css */
+.hero-section{
+    display: grid;
+    grid-template-columns: 1fr 5fr;
+    margin: 2rem 2rem;
+}
+
+/* Acheivements Fetured Category CSS */
+
+
```

Comment:
suggestion: maintain equal spaces for readability and clean code structure.

Reply:
Done